src/main/java/com/rsmart/kuali/kfs/cr/CRConstants.java

```diff
@@ -104,6 +104,5 @@
     public static final String SRC_NOT_FOUND = "CR_SOURCE_FOR_NOT_FOUND";
     public static final String BNK_CD_NOT_FOUND = "CR_NOT_FOUND_BANK_CD";
 
-    public static final String BC_REPORT_TEMPLATE_CLASSPATH = "org/kuali/kfs/module/bc/report/";
-    public static final String BC_REPORT_MESSAGES_CLASSPATH = BC_REPORT_TEMPLATE_CLASSPATH + "BudgetOrgReport";
+    public static final String CHECK_RECONCILIATION_REPORT_MESSAGES_CLASSPATH = "com/rsmart/kuali/kfs/cr/report/CRReport";
```

Comment:
Did you need this new one, or could you just use the existing REPORT_MESSAGES_CLASSPATH constant?